cmd/chart/upgrade.go

```diff
@@ -106,6 +106,10 @@ Otherwise, it returns a non-zero exit code and the updated values.yaml file.`,
 
 			if latestTag != tag {
 				updated++
+				// Semver is "eating" the "v" prefix, so we need to add it back, if it was there in first place
+				if strings.HasPrefix(tag, "v") {
```

Comment:
There could be other prefixes or suffixes, it might be more robust if there was a map kept so the tag could be restored.

Reply:
Happy to add new patterns in the future!